.travis.yml

```diff
@@ -15,9 +15,10 @@ jobs:
   include:
     - if: type = pull_request
       name: checkpatch
-      script:
+      script: # FIXME: replace with TRAVIS_COMMIT_RANGE or similar
```

Comment:
?? why is this relevant for this specific PR - or what did you mean to day? Looks like a different issue,

Reply:
It is a different issue but it is (or at least: was) a bit related because you don't want to checkpatch the cherry-pick. I'll remove it.